portal/common/lib/crypto.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+const { subtle } = globalThis.crypto;
+
+/**
+ * Calculates SHA-256 hash of input message.
+ * @param message ArrayBuffer to hash
+ * @returns Promise<Uint8Array> Resulting hash as Uint8Array
+ */
+export async function sha256(message: ArrayBuffer) {
```

Comment:
Nit: unsure about this, but shouldn't the return type be specified?
```suggestion
export async function sha256(message: ArrayBuffer): Promise<Uint8Array> {
```